packages/desktop/lib/electron/managers/transak.manager.ts

```diff
@@ -95,6 +95,11 @@ export default class TransakManager implements ITransakManager {
             windows.transak.webContents.send('transak-data', _data)
         })
 
+        windows.transak.webContents.setWindowOpenHandler(({ url }) => {
+            void shell.openExternal(url)
+            return { action: 'deny' }
```

Comment:
Do I understand it correctly that you're overriding the window.open() function to open the window in the browser instead of the electron process?

Reply:
Yes